tartiflette/sdl/sdl_cli.py

```diff
@@ -0,0 +1,35 @@
+from tartiflette.sdl.builder import transform_ast_to_schema, \
```

Comment:
If is doesn't fit in one line, prefers multiple lines
```python
from tartiflette.sdl.builder import transform_ast_to_schema
from tartiflette.sdl.builder imprt parse_graphql_sdl_to_ast
```

Reply:
Recommended format is PEP328 (which uses parenthesis). In any case, I will have to fix this later as this is the default syntax by PyCharm and will be updated on each new save and/or import so if I change it now it will be overridden.

Reply:
ok